### DISCUSSION TITLE
examples/apps/ai_livertumor_seg_app

### DISCUSSION THREAD

**Discussion by SameerShanbhogue** [Project-MONAI/monai-deploy-app-sdk#451] [General]
Hi

Where can i find model.ts for examples/apps/ai_livertumor_seg_app. link https://ngc.nvidia.com/catalog/models/nvidia:med:clara_pt_liver_and_tumor_ct_segmentation  provided in notebook does not exist.

**Comment by MMelQin** [comment]
This particular model has been removed from NGC, however, a version of it is embedded in a published MAP container image, ghcr.io/mmelqin/monai_ai_lung_seg_app:1.0. Please pull the image and copy the model from within the image, at /opt/monai/models, for this version. For the published tutorial notebook, steps are provided to download and unzip the file which contains the model, if applicable, as well as the test data.
